components/hostSignup.tsx

```diff
@@ -1,24 +1,125 @@
-import { useState } from 'react';
-import { useForm } from 'react-hook-form';
-import { Space } from '@mantine/core';
+import { useForm } from '@mantine/hooks';
+
+import {
+  Space,
+  Paper,
+  Title,
+  TextInput,
+  Textarea,
+  Button,
+  NumberInput,
+  MultiSelect,
+  Checkbox,
+} from '@mantine/core';
+import { MdOutlineMailOutline, MdMap, MdPhone } from 'react-icons/md';
+import axios from 'axios';
+
+const languagesOptions = [
+  { value: 'English', label: 'English' },
+  { value: 'Ukrainian', label: 'Ukrainian' },
+  { value: 'Russian', label: 'Russian' },
+  { value: 'Polish', label: 'Polish' },
+  { value: 'Romanian', label: 'Romanian' },
+  { value: 'German', label: 'German' },
+  { value: 'Hungarian', label: 'Hungarian' },
+];
+
 export const HostSignup = () => {
-  const { register, handleSubmit } = useForm();
-  const [data, setData] = useState('');
+  const form = useForm({
+    initialValues: {
+      firstName: '',
+      lastName: '',
+      phoneNumber: '',
+      email: '',
+      cityRegion: '',
+      accomodationDetails: '',
+      hostCapacity: 0,
+      languages: '',
+      termsOfService: false,
+    },
+
+    validationRules: {
+      email: (value) => /^\S+@\S+$/.test(value),
+    },
+  });
+
+  const onSubmitHandler = (values: typeof form['values']) => {
+    console.log(values);
+    axios({ method: 'POST', url: '/api/host', data: values });
+  };
 
   return (
     <>
       <Space h="sm" />
-      <iframe
-        className="airtable-embed"
-        src="https://airtable.com/embed/shrGJ9QFefvxi47xL?backgroundColor=yellow"
-        frameBorder="0"
-        style={{
-          width: '100%',
-          height: '100%',
-          background: 'transparent',
-          display: 'block',
-        }}
-      ></iframe>
+      <Paper padding="lg" shadow="sm" radius="md" withBorder>
+        <Title order={3}>Find a host</Title>
+        <Space h="lg" />
+
+        <form onSubmit={form.onSubmit(onSubmitHandler)}>
```

Comment:
Should we just use the session.user.name from NextAuth for this instead of having them fill it out?